### ISSUE TITLE
Update "out of 0781 scope" pages in 526

### ISSUE THREAD

**Issue by jinaryu** [department-of-veterans-affairs/va.gov-team#89780]
Ticket to track recs for out of scope pages (pages in 526, but not in 0781)

https://www.figma.com/design/r3Aj9FtLFS989mlVeBsgJg?node-id=4546-69555#866651165
Update 4142 (current one in 526 is outdated)
https://dsva.slack.com/archives/C05MPTGMX0E/p1720453920280419

**Comment by jinaryu** [comment]
[Review page ](https://dsva.slack.com/archives/C01DBGX4P45/p1727191290238349)could be updated

**Comment by jinaryu** [comment]
@lisacapaccioli tagging you for awareness

I had initially created this ticket earlier in the 0781 design process, when we didn't want to make changes to other steps of the form, namely evidence. However, after concept testing and realizing there really wasn't a good way to do this without touching the evidence section of the 526, we decided to include the evidence changes as a part of the 0781 redesign. 

Takeaways 

- Ignore the points about evidence and 4142. We decided to update those
- "Additional disability benefits" and the forms included in that section of the 526 are good candidates of forms to move into this new additional forms step
